apis/v1alpha1/pmap_event.pb.go

```diff
@@ -118,11 +118,17 @@ type GitHubSource struct {
 	unknownFields protoimpl.UnknownFields
 
 	// Required. The repository name where the payload is located.
-	RepoName string `protobuf:"bytes,1,opt,name=repo_name,json=repoName,proto3" json:"repo_name,omitempty"`
+	RepoName string `protobuf:"bytes,1,opt,name=repo_name,json=repoName,proto3" json:"repo-name,omitempty"`
```

Comment:
https://pkg.go.dev/encoding/json  should use `json:"repoName,omitempty"` to follow the naming convention of json tag

Reply:
Good catch. Aren't these generated?